### PR TITLE
Add job description for Senior Developer and Senior product UX designer 

### PR DESCRIPTION
**Pull request by siapankina**
Added 2 job descriptions:
-Senior Developer 
-Senior product UX designer

### CODE REVIEW

**Comment by ubxd** [comment]
Deploy preview for *unboxed-co-production* ready!

Built with commit 746f27be2a61bb17bc102fe33d0209ff95b6f9da

https://deploy-preview-643--unboxed-co-production.netlify.app

**Comment by ubxd** [comment]
Deploy preview for *unboxed-co-production* ready!

Built with commit 74b23171e6ecd156069c2442e45c6ff0695e0330

https://deploy-preview-643--unboxed-co-production.netlify.app

File: source/jobs.html.erb [hunk 1]
```diff
@@ -20,7 +20,15 @@ unboxed_meeting_image:
...
-  content:
+  content: 
+  - title: Senior product designer
+    name: Senior product designer
+    text: "We are on the lookout for a London-based Senior Product Designer, with solid experience of designing user flows and prototypes, who loves working with teams to turn designs into digital products that make a meaningful impact in the world."
+    link: /jobs/senior-product-designer
+  - title: Senior developer
+    name: Senior developer
+    text: "We are on the lookout for Ruby on Rails Developers to join us in London. As a core part of our team, our developers are responsible for building scalable, sustainable systems from project inception right through to production."
+    link: /jobs/developer
```
Comment (cdccollins) [inline source/jobs.html.erb]: This should link to the senior developer job. I think there's a file in there called developer_senior_developer or something like that. Probably best to edit that to make it just a senior developer job role.

**Comment by cdccollins** [comment]
Could you also make the title of this PR more specific or add a description saying exactly which ones you've done please? It makes it easier to find work later on 😄

File: source/jobs/senior-product-designer.html.erb [hunk 2]
```diff
@@ -8,7 +8,7 @@ header:
   image: "/jobs/images/twitter-card-image-design.jpg"
 
 introduction:
-  title: "This vacancy is no longer open for applications"
+  title: "Are you a Senior Digital Product Designer?"
```
Comment (cdccollins) [inline source/jobs/senior-product-designer.html.erb]: I think this is the wrong file? Isn't there a senior ux designer file in there somewhere?

Comment (borisdivjak) [reply source/jobs/senior-product-designer.html.erb]: > I think this is the wrong file? Isn't there a senior ux designer file in there somewhere?

Yes, this is not the right file ... I made a comment on Slack about it yesterday:

> sorry, I think I’ve linked to the wrong job page. It should be this one: https://deploy-preview-643--unboxed-co-production.netlify.app/jobs/senior-product-ux-designer

Comment (cdccollins) [reply source/jobs/senior-product-designer.html.erb]: I'm a bit confused. There are two job ads - senior product designer, and senior ux designer. It now links to the correct one, but why has this page (senior product designer) been edited if it's not the one we're using.

Comment (borisdivjak) [reply source/jobs/senior-product-designer.html.erb]: It might be because I posted a link to it on Slack initially, but that was a mistake, sorry.

Comment (borisdivjak) [reply source/jobs/senior-product-designer.html.erb]: You can delete the 'senior product designer' one. And just keep the 'senior product (ux) designer'.

Comment (siapankina) [reply source/jobs/senior-product-designer.html.erb]: Removed all changes from that file

**Review by cdccollins — APPROVED** [review]
This looks good 😄 . Can you please squash the commits before merging

**Pull request merged**